mapbox_tilesets/scripts/cli.py

```diff
@@ -216,6 +216,38 @@ def status(tileset, token=None, indent=None):
     click.echo(json.dumps(r.json(), indent=indent))
 
 
+@cli.command("tilejson")
+@click.argument("tileset", required=True, type=str)
+@click.option("--token", "-t", required=False, type=str, help="Mapbox access token")
+@click.option("--indent", type=int, default=None, help="Indent for JSON output")
+def tilejson(tileset, token=None, indent=None):
+    """View the TileJSON of a particular tileset.
+    Can take a comma-separated list of tilesets for a composited TileJSON.
+
+    tilesets publish <tileset_id>,<tileset_id>
+    """
+    mapbox_api = _get_api()
+    mapbox_token = _get_token(token)
+
+    # validate tilesets by splitting comma-delimted string
+    # and rejoining it
+    print(tileset)
+    tilesets = []
+    for t in tileset.split(","):
+        if not utils.validate_tileset_id(t):
+            raise errors.TilesetNameError(t)
+        tilesets.append(t)
```

Comment:
I don't think you need to build a new copy of `tileset.split` since the command will error out rather than request tilejson for any invalid tileset id. You could do something like
```py
for t in tileset.split(","):
        if not utils.validate_tileset_id(t):
            raise errors.TilesetNameError(t)
url = "{0}/v4/{1}.json?access_token={2}".format(
        mapbox_api, tilesets, mapbox_token
    )
```
And skip a round of string -> array -> string

Reply:
Good catch - updated in https://github.com/mapbox/tilesets-cli/pull/62/commits/0eab58cc66234b0a5ec3cac2b5d03a89c95418a6